audio/data.py

```diff
@@ -355,3 +379,9 @@ def from_folder(cls, path: PathOrStr = '.', extensions: Collection[str] = None,
         if not extensions:
             extensions = AUDIO_EXTENSIONS
         return cls(get_files(path, extensions, recurse), path, **kwargs)
+
+def open_audio(fn:PathOrStr, after_open:Callable=None)->AudioItem:
+    if not str(fn).lower().endswith(AUDIO_EXTENSIONS): raise Exception("Invalid audio file")
+    sig, sr = torchaudio.load(fn)
+    if after_open: x = after_open(sig, sr)
```

Comment:
Interesting feature but is it really needed?

Reply:
I like it and think we'll need the pattern anyway, it's opening the door to callbacks right?